src/content/reference/manifest.mdx

```diff
@@ -254,6 +254,21 @@ deploy:
 - `virtualService`: the name of the virtual service you want to divert.
 - `hosts`: the list of hosts you want to divert in the developer namespace. Requests to these virtual services will inject the header `x-okteto-divert`.
 
+#### Deploy remotely
+
+Remote deploy is useful to share the same tools and configuration across the team.
+
+This will provide all the members in the team the same deployment experience since they are sharing the config and deploy tools defined in the image provided.
+
+```yaml
+deploy:
+  image: okteto/pipeline-installer:1.7.6
+  commands:
+    - helm upgrade --install movies chart --set api.image=${OKTETO_BUILD_API_IMAGE} --set frontend.image=${OKTETO_BUILD_FRONTEND_IMAGE}
+```
+
+> If no image is provided the command will run in the local machine using the local tools and configuration
```

Comment:
I find this odd. If I'm passing a `--remote` flag, I'm expecting this to run remotely. 

could we instead:
a) fail the command
b) use a default image (the same that the installer perhaps?)

cc @omnomagonz what do you think?

Reply:
If image is set in the manifest, Okteto deploy command will run on remote.
If image is **not** set in the manifest, Okteto deploy command will run locally.
If image is **not** set in the manifest but the user uses the `--remote` flag, Okteto deploy command will run on remote using the pipeline installer image.

Reply:
@jLopezbarb can we link to the repo with the default image? (or document it somehow?)

```suggestion

If you define an image in the okteto manifest, `okteto deploy` will always run in remote mode.  If you don't define an image, and you run in remote mode, okteto will use a default container image.
```

Reply:
(whatever text we end up with here, we should also apply to `destroy`.